src/Query/Postgresql/IfNull.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace DoctrineExtensions\Query\Postgresql;
+
+use Doctrine\ORM\Query\AST\ArithmeticExpression;
+use Doctrine\ORM\Query\AST\Functions\FunctionNode;
+use Doctrine\ORM\Query\Lexer;
+use Doctrine\ORM\Query\Parser;
+use Doctrine\ORM\Query\QueryException;
+use Doctrine\ORM\Query\SqlWalker;
+
+class IfNull extends FunctionNode
+{
+    /** @var ArithmeticExpression */
+    private $expr1;
+
+    /** @var ArithmeticExpression */
+    private $expr2;
+
+    /**
+     * @param Parser $parser
+     *
+     * @throws QueryException
+     */
+    public function parse(Parser $parser)
+    {
+        $parser->match(Lexer::T_IDENTIFIER);
+        $parser->match(Lexer::T_OPEN_PARENTHESIS);
+        $this->expr1 = $parser->ArithmeticExpression();
+        $parser->match(Lexer::T_COMMA);
+        $this->expr2 = $parser->ArithmeticExpression();
+        $parser->match(Lexer::T_CLOSE_PARENTHESIS);
+    }
+
+    /**
+     * @param SqlWalker $sqlWalker
+     *
+     * @return string
+     */
+    public function getSql(SqlWalker $sqlWalker): string
+    {
+        return sprintf(
+            'IFNULL(%s, %s)',
```

Comment:
I think PostgreSQL does not have `IFNULL` function. Did you mean [`COALESCE`](https://www.postgresql.org/docs/9.3/functions-conditional.html#FUNCTIONS-COALESCE-NVL-IFNULL)?

Reply:
Postgres has the function https://www.postgresql.org/docs/9.2/functions-conditional.html